src/invidious/channels/about.cr

```diff
@@ -70,7 +71,9 @@ def get_about_info(ucid, locale) : AboutChannel
     # if banner.includes? "channels/c4/default_banner"
     #  banner = nil
     # end
+    author_verified_badges = initdata["header"]?.try &.["c4TabbedHeaderRenderer"]?.try &.["badges"]?
 
+    author_verified = (author_verified_badges && author_verified_badges.size > 0)
```

Comment:
I see that you copied this from the example I gave you in the discussions. However, you forgot to use `.as_a` on `author_verified_badges`, and you also forgot to check if the badges array has a "verified" badge child.